src/modules/site/base/utils/auth.py

```diff
@@ -15,11 +20,22 @@
                                 get_jwt_identity,
                                 get_current_user,
                                 verify_jwt_in_request,
-                                jwt_required)
+                                jwt_required,
+                                decode_token)
 
 from caendr.models.datastore import User
+from caendr.models.datastore.user_token import UserToken
 from extensions import jwt
 
+def create_one_time_token(id, destination='/'):
+  expires_delta = timedelta(seconds=JWT_LIMITED_TOKEN_EXPIRES)
```

Comment:
consider prefixing the ENV name with 'PASSWORD_RESET_' to avoid confusing it with the regular JWT session token